prql-elixir/test/prql_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule PRQLTest do
+  use ExUnit.Case
+  doctest PRQL
+
+  @compile_opts [signature_comment: false]
+
+  test "compiles PRQL" do
+    prql_query = """
+      from customers
+    """
+
+    excepted_result = "SELECT\n  *\nFROM\n  customers"
+
+    assert PRQL.compile(prql_query, @compile_opts) == {:ok, excepted_result}
+  end
+
+  test "return errors on invalid query" do
+    excepted_result =
+      "{\"inner\":[{\"reason\":\"Unknown name invalid\",\"hint\":null,\"span\":{\"start\":0,\"end\":7},\"display\":\"Error: \\n   ╭─[:1:1]\\n   │\\n 1 │ invalid\\n   · ───┬───  \\n   ·    ╰───── Unknown name invalid\\n───╯\\n\",\"location\":{\"start\":[0,0],\"end\":[0,7]}}]}"
```

Comment:
To the extent elixir has multiline strings, this would be much easier to read as one! (but no stress)

Reply:
Sure i will add multi lines

Reply:
Its bit difficult to do with new lines, shall we do it later?

Reply:
Yes completely fine